thread/common/threadhelpers.cpp

```diff
@@ -134,11 +130,10 @@ omrthread_spinlock_acquire(omrthread_t self, omrthread_monitor_t monitor)
 #endif /* OMR_THR_JLM */
 
 #if defined(OMR_THR_SPIN_WAKE_CONTROL)
-	if (spinning) {
-		VM_AtomicSupport::subtract(&monitor->spinThreads, 1);
-	}
+	VM_AtomicSupport::subtract(&monitor->spinThreads, 1);
 #endif /* defined(OMR_THR_SPIN_WAKE_CONTROL) */
 
+exit:
```

Comment:
The `exit` label is unconditionally compiled in but only used under `OMR_THR_SPIN_WAKE_CONTROL` - does this compile successfully if the #define isn't set?

Reply:
no, it will not compile properly if the label is unused. moved `exit:` label within `OMR_THR_SPIN_WAKE_CONTROL` ifdefs.